nornir_nautobot/plugins/tasks/dispatcher/default.py

```diff
@@ -147,7 +148,13 @@ def compliance_config(
 
     @staticmethod
     def generate_config(
-        task: Task, logger, obj, jinja_template: str, jinja_root_path: str, output_file_location: str
+        task: Task,
+        logger,
+        obj,
+        jinja_template: str,
+        jinja_root_path: str,
+        output_file_location: str,
+        jinja_filters: Optional[dict],
```

Comment:
This will require a default parameter, otherwise anywhere that it is is called without `jinja_filters` will have an error.

```suggestion
        jinja_filters: Optional[dict]=None,
```

Reply:
Yes. Totally missed that. Thanks @jvanderaa